tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "inlineSources": true,
     "preserveConstEnums": true,
     "removeComments": false,
-    "sourceMap": true
+    "sourceMap": true,
+    "esModuleInterop": true
```

Comment:
This is usually an indication something is wrong (prob the dockerode stuff) 😅 
We should not enable it unless absolutely necessary.